crates/optimism/node/src/node.rs

```diff
@@ -65,14 +66,44 @@ use std::{marker::PhantomData, sync::Arc};
 pub trait OpNodeTypes:
     NodeTypes<Payload = OpEngineTypes, ChainSpec = OpChainSpec, Primitives = OpPrimitives>
 {
+    /// The pooled transaction type for the pool.
+    type OpPooledTx: EthPoolTransaction<Consensus = TxTy<Self>, Pooled = Self::OpNetworkPooledTx>
+        + OpPooledTx;
+    /// The network pooled transaction type.
```

Comment:
helpful to be a bit more specific that it's not over rpc with "as broadcasted over gossipsub p2p" or smthg like that

Reply:
nope, now it introduces a helper trait to refactor bounds